WcaOnRails/app/models/country.rb

```diff
@@ -24,10 +24,19 @@ def self.find_by_iso2(iso2)
     c_all_by_id.values.select { |c| c.iso2 == iso2 }.first
   end
 
-  COMPARE_LOCALIZED_NAMES = lambda do |a, b|
-    # We transliterate names so that country/continents starting with accent
-    # don't end up at the very bottom of the list because of the accents
-    I18n.transliterate(a.first) <=> I18n.transliterate(b.first)
+  # Comparison function may depend on the locale, so we use an array of lambdas,
+  # defaulting to the latin transliterate+compare
+  COMPARE_LOCALIZED_NAMES = Hash.new do |hash, key|
```

Comment:
I have an idea!
Something like that here
```ruby
COMPARE_LOCALIZED_NAMES = lambda do |locale, a, b|
  # stuff...
end
```
And that's how a call would look like
```ruby
array.sort! &Country::COMPARE_LOCALIZED_NAMES.curry[l]
```
Basically, we pass a parameter to the function =)